library/src/main/java/io/github/wax911/library/annotation/processor/GraphProcessor.kt

```diff
@@ -73,11 +78,28 @@ class GraphProcessor private constructor(assetManager: AssetManager?) {
                     }
                 }
             }
+
+            patchQueries()
         } catch (e: IOException) {
             e.printStackTrace()
         }
     }
 
+    /**
+     * Patch any query with fragment references, that don't already have the fragment defined with the query.
+     */
+    @Synchronized
+    private fun patchQueries() {
+        _graphFiles.entries
+            .filter { GraphRegexUtil.containsAQuery(it.value) }
```

Comment:
Oh, I also added a new regex to test whether the current file/graph contents have a query. We really only want to "patch" queries, but this `_graphFiles` map contains queries and fragments.